lib/plurimath/math/function/menclose.rb

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 
 require_relative "binary_function"
+require_relative "../../mathml/utility"
 
 module Plurimath
   module Math
     module Function
       class Menclose < BinaryFunction
```

Comment:
Similar to my comment above whether this is generic or MathML specific?

Reply:
This class itself is not generic but other conversions output are supported.